README.md

```diff
@@ -16,11 +16,109 @@ var backlight = require('rpi-backlight');
 
 backlight.powerOn();
 backlight.powerOff();
+// 1 = Screen is OFF
+// 0 = Screen is ON
+backlight.getActualPowerValue();
+
+backlight.setMaxBrightness();
+backlight.getActualBrightness();
+// The screen goes Off at <= 9 brightness value
+backlight.setBrightnessValue(value);
+
+// Work in progress
+backlight.goToBrightnessValue(toValue, speed);
+
 ```
 
-## Coming soon
+## Forked from
+
+- [pimterry/rpi-backlight](https://github.com/pimterry/rpi-backlight)
+
+## Example of a simple app
+
+index.js:
+```javascript
+const express = require('express');
+const app = express();
+const backlight = require('./rpi-backlight/index');
```

Comment:
For this example, since it's using this module, it should be `require('rpi-backlight')`